projects/1.16/assets/wither-totem/totemmod/lang/zh_cn.json

```
@@ -1 +1,8 @@
-{}
\ No newline at end of file
+{
+    "block.totemmod.totem_base": "图腾柱脚",
+    "block.totemmod.totem_top": "凋灵骷髅图腾",
+    "effect.totemmod.looting": "抢夺",
+    "block.totemmod.tooltip_1": "一个完整建造的图腾",
+    "block.totemmod.tooltip_2": "会给你抢夺的效果，",
+    "block.totemmod.tooltip_3": "但是也会给附近的怪物此效果。"
```

```suggestion
    "block.totemmod.tooltip_3": "但是也会强化附近的怪物。"
```
`also buffs monsters nearby`..?
而且给附近的怪物抢夺效果有点怪